addons/captives/CfgVehicles.hpp

```diff
@@ -16,7 +16,7 @@ class CfgVehicles {
                 class ACE_RemoveHandcuffs {
                     displayName = CSTRING(ReleaseCaptive);
                     selection = "righthand";
-                    distance = 2;
+                    distance = HANDCUFFS_DISTANCE;
```

Comment:
not really related to this pr, but just noting that because it's under main_actions these `distance`s have no effect

Reply:
Sure about that? I remember the handcuffs requiring you to be closer than the main interaction point to show up, but I can't get ingame to test rn.

Reply:
Oh, I see what you mean now. Nevermind.